authorization/1.6/apidocs/allclasses-index.html

```diff
@@ -0,0 +1,202 @@
+<!DOCTYPE HTML>
+<!-- NewPage -->
+<html lang="en">
+<head>
+<!-- Generated by javadoc (12.0.1) on Sun Jul 14 22:30:54 CEST 2019 -->
```

Comment:
Minor issue, but some JavaDocs are generated with JDK 8, others with JDK 11 or 12. Won't this generate inconsistent stylings?

Btw: I particularly like JDK 11+ JavaDoc as it contains the search funcionality

Reply:
@ggam The combined platform javadocs are regenerated from source.

Reply:
Good remark though @ggam, I was actually noticing that as well, but indeed the combined docs are regenerated.

There's a couple of other small inconsistencies, like the pdf files for the spec sometimes have -spec in the file name and sometimes not.